<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connect</class>
 <widget class="QWidget" name="Connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>575</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>771</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="lblOutput">
        <property name="text">
         <string>Program Output</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QListWidget" name="lstOutput">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblScan">
        <property name="text">
         <string>Scanning ...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cmbIpAdd"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblIpAdd">
        <property name="text">
         <string>IP Addresses</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnScan">
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <widget class="QPushButton" name="btnDontListen">
        <property name="text">
         <string>Don't Listen</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblListening">
        <property name="text">
         <string>Listening mode ON.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnListen">
        <property name="text">
         <string>Listen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
